requirements_dev.txt

```diff
@@ -0,0 +1 @@
+pytest
```

Comment:
can you pin specific versions for all the requirements? It's much easier to avoid a hassle in the future when breaking changes happen.